ui/raidboss/data/06-ew/raid/p4s.ts

```diff
@@ -473,6 +790,25 @@ const triggerSet: TriggerSet<Data> = {
       netRegexJa: NetRegexes.startsUsing({ id: '6A2D', source: 'ヘスペロス', capture: false }),
       response: Responses.aoe(),
     },
+    {
+      id: 'P4S Act',
```

Comment:
Is this for the other PR?

Reply:
It is, but it does work and otherwise is not used in this PR.